python/gym_jiminy/rllib/gym_jiminy/rllib/utilities.py

```diff
@@ -368,7 +369,7 @@ def train(algo: Algorithm,
           logdir: Optional[Logger] = None,
           checkpoint_period: int = 0,
           verbose: bool = True,
-          debug: bool = False) -> str:
+          debug: bool = False) -> _TrainingResult:
```

Comment:
I already implemented a patch in my other PR. Maybe you could backport what I did here. I added some extra details.

Reply:
Ok, I reverted this change and renamed the PR.